tests/fsharpqa/Source/CodeGen/EmittedIL/Tuples/OptionalArg01.il.bsl

```diff
@@ -205,7 +205,7 @@
   {
     // Code size       7 (0x7)
     .maxstack  8
-    .line 19,19 : 5,11 ''
+    .line 10,10 : 23,44 ''
```

Comment:
These changes are from the cases we expect, here C.F is being inlined and we now have breakpoints for that:
```fsharp
// A code+optimization pattern, see https://github.com/Microsoft/visualfsharp/issues/6532
type C() = 
    static member inline F (?x1: A, ?x2: A) = 
        let count = 0
        let count = match x1 with None -> count | Some _ -> count + 1
        let count = match x2 with None -> count | Some _ -> count + 1
        let attribs = ResizeArray<_>(count)
        match x1 with None -> () | Some v1 -> attribs.Add(v1)
        match x2 with None -> () | Some v2 -> attribs.Add(v2)
        attribs

//Expect rough equivalent of:
//        let d = ResizeArray<_>(0)
//        d
let test() = 
    C.F ()

```